relationships/synthesis/INFRA-NR_DOCKER_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,40 @@
+relationships:
+  - name: nrInfraDockerContainerEXTService
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "ContainerSample" ]
+      - attribute: entity.type
+        anyOf: ["CONTAINER"]
+      - attribute: container.id
+        present: true
+    relationship:
+      expires: P75M
+      relationshipType: HOSTS
+      source:
+        buildGuid:
+          account:
+            attribute: accountId
+          domain:
+            value: INFRA
+          type:
+            value: CONTAINER
+            valueInGuid: NA
+          identifier:
+            fragments:
+              - attribute: container.id
+            hashAlgorithm: FARM_HASH
+      target:
+        buildGuid:
+          account:
+            attribute: accountId
+          domain:
+            value: EXT
+          type:
+            value: SERVICE
+          identifier:
+            fragments:
+              - attribute: service.name
```

Comment:
Just wondering - will the service.name on a container sample reflect the name of a related EXT-SERVICE entity ? 🤔

Reply:
Could it be the other way around, that EXT-SERVICE telemetry may tell us the container id ?